### PR TITLE
update my bio and links

### PR DESCRIPTION
**Pull request by shivam-sharma7**
update my bio and links

 
  

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4483"></a>

### CODE REVIEW

**Comment by shivam-sharma7** [comment]
review/ @Pradumnasaraf

File: data/shivam-sharma7.json [hunk 1]
```diff
@@ -19,6 +19,12 @@
       "name": "Blog's",
```
Comment (priyankarpal) [inline data/shivam-sharma7.json:19]:
``` suggestion
      "name": "Blog",
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]
fix this spelling

File: data/shivam-sharma7.json [hunk 2]
```diff
@@ -16,9 +16,15 @@
     },
 
     {
-      "name": "Blog's",
+      "name": "Blog",
       "url": "https://shivam-sharma.hashnode.dev/",
       "icon": "FaLink"
```
Comment (priyankarpal) [inline data/shivam-sharma7.json:21]: Add Hashnode icon
``` suggestion
      "icon": "SiHashnode"
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻

**Comment by Pradumnasaraf** [comment]
> review/ @Pradumnasaraf

Hey, you need to ping any specific maintainers. Anyone can review and merge

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**